demo/src/app/components/typeahead/overview/sections/introduction/template.html

```diff
@@ -0,0 +1,31 @@
+<ng-template
+  #displayResult
+  let-color="result"
+  let-term="term"
+>
+  <div class="middle">
+    <div class="color-chip" [ngStyle]="{'background-color': '#' + color.hexCode}"></div>
+    <ngb-highlight [result]="color.name" [term]="term"></ngb-highlight><span> ({{color.hexCode}})</span>
+  </div>
+</ng-template>
+
+<p>Typeahead example: </p>
+
+<input
```

Comment:
Maybe limit width and put in one line, like so:

```
Pick a color: [                    ]
```

Now it takes the whole width which is strange

Reply:
Totally agreed